Ref: Ionic Docs on Capacitor and Cordova (#5296)
getsentry/sentry-docs: src/platforms/javascript/guides/cordova/ionic.mdx

---
title: Ionic
sidebar_order: 201
noindex: true
redirect_from:
  - /clients/cordova/ionic/
description: "Learn how to use Sentry with Ionic."
---

<Alert level="warning" title="Warning">

> It is highly recommended to use Ionic with Capacitor, _click here_ to be redirected to the new documentation of Ionic with Sentry Capacitor.
> </Alert>

Reviewer: ```suggestion
We strongly recommended that you use Ionic with Capacitor. Check out our [Ionic with Sentry Capacitor documentation](/platforms/javascript/guides/capacitor/ionic/.

</Alert>
```

Author: Oh thanks for the comment, I was looking on how to properly link a page from another framework

---
title: Ionic
sidebar_order: 201
noindex: true
notoc: true
redirect_from:
  - /clients/cordova/ionic/
description: "Learn how to use Sentry with Ionic."
---

<Alert level="warning" title="Warning">

> It is highly recommended to use Ionic with Capacitor, _click here_ to be redirected to the new documentation of Ionic with Sentry Capacitor.

Author: ```suggestion
We strongly recommended that you use Ionic with Capacitor. Check out our [Ionic with Sentry Capacitor documentation](/platforms/javascript/guides/capacitor/).

```

---
title: Ionic
sidebar_order: 201
noindex: true

> notoc: true

Reviewer: For some reason this page is still showing up in the TOC in the preview. I'm going to pull this branch into my local and test it.

> notoc: true

Reviewer: ```suggestion
notoc: true
draft: false
```

Author: Oh I didn't notice you changed that flag, I'll rollback to only alter the draft: false.
